src/client/Microsoft.Identity.Client/UserAssertion.cs

```diff
@@ -51,7 +51,7 @@ public UserAssertion(string assertion, string assertionType)
                 throw new ArgumentNullException(nameof(assertionType));
             }
 
-            var crypto = PlatformProxyFactory.CreatePlatformProxy(null).CryptographyManager;
+            PlatformsCommon.Interfaces.ICryptographyManager crypto = PlatformProxyFactory.CreatePlatformProxy(null).CryptographyManager;
```

Comment:
using statement here

Reply:
Isn't `ICryptographyManager` not implementing `IDisposable` @gladjohn?